.github/workflows/image-manual.yml

```diff
@@ -56,8 +56,8 @@ jobs:
       - run: |
           # build, run tests and if all worked publish a custom image
           export PROJECT_TAG="${DOCKER_TAG}"
-          # Use {GITHUB_SHA::8} for shorter SHA
-          export VERSION="${GITHUB_SHA}"
+          # The version of skupper-router should be the same as the version of the image.
+          export VERSION="${DOCKER_TAG}"
```

Comment:
It would be nice IMO to keep first 6 or 8 characters of the commit hash in the version. But that is for separate enhancement. It might also clash with Python versioning, so https://github.com/skupperproject/skupper-router/issues/973 would need to be looked into at the same time